hail_scripts/reference_data/combine.py

```diff
@@ -167,13 +146,9 @@ def update_existing_joined_hts(
 ):
     joined_ht = hl.read_table(destination_path)
     dataset_ht = get_ht(dataset, genome_version)
-    if 'coverage' not in dataset:
-        joined_ht = joined_ht.drop(dataset)
-        joined_ht = joined_ht.drop(f'{dataset}_globals')
-        joined_ht = joined_ht.join(dataset_ht, 'outer')
-        joined_ht = joined_ht.filter(
-            hl.any([~hl.is_missing(joined_ht[dataset]) for dataset in datasets]),
-        )
-    else:
-        joined_ht = annotate_coverage(joined_ht, dataset_ht, dataset)
+    joined_ht = joined_ht.drop(dataset, f'{dataset}_globals')
```

Comment:
huh, I did not know you could drop row annotations and globals at the same time, neat

Reply:
hm... I'm actually not sure this has been tested.. Also not sure why I decided to combine the `drop` lines.

Reply:
I'll test real quick.

Reply:
Yep!

```
In [7]: sample_lookup_ht.describe()
----------------------------------------
Global fields:
    'updates': set<struct {
        callset: str,
        project_guid: str
    }>
----------------------------------------
Row fields:
    'id': int32
    'ref_samples': dict<str, set<str>>
    'het_samples': dict<str, set<str>>
    'hom_samples': dict<str, set<str>>
----------------------------------------
Key: ['id']
----------------------------------------

In [8]: sample_lookup_ht.drop('ref_samples', 'updates').describe()
----------------------------------------
Global fields:
    None
----------------------------------------
Row fields:
    'id': int32
    'het_samples': dict<str, set<str>>
    'hom_samples': dict<str, set<str>>
----------------------------------------
Key: ['id']
----------------------------------------
```

I think I probably have been through this exercise before and don't have a memory of it.

Reply:
Sweet!